docs/tasks/tls/managing-tls-in-a-cluster.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Managing TLS Certificates in a Cluster
+assignees:
+- mikedanese
+- beacham
+- liggit
+---
+
+* TOC
+{:toc}
+
+## Overview
+
+Every Kubernetes has a cluster root Certificate Authority (CA). The CA is
+generally used by cluster components to validate the server certificate of the
```

Comment:
"generally" not necessary.
"the server certificate of the API server" -> "the API server's certificate"